src/sample/direct.jl

```diff
@@ -47,14 +49,14 @@ end
 
 function set_clock!(dc::DirectCall{T}, clock::T, distribution::Exponential,
         enabled::Symbol, rng::AbstractRNG) where {T}
-    if Base.:(==)(enabled, :Enabled)
+    if enabled == :Enabled
```

Comment:
Ah! I was fighting with this in a previous Julia version. Nice that it's fixable!

Reply:
Yes, apparently not a problem after >= 1.6